### DISCUSSION TITLE
A For You recommendation apparently shows up "because I starred it" but I didn't

### DISCUSSION THREAD

**Discussion by softorangetech1122** [community/community#32364] [Other Feature Feedback, Questions, & Ideas] [answered]
I didn't star fontsource/fontsource, but apparently I did...

![Screen Shot 2022-09-08 at 7 48 07 PM](https://user-images.githubusercontent.com/111031232/189256197-29f0a692-3b1d-4813-988f-3e3bf179b2c8.png)

And proof...

![Screen Shot 2022-09-08 at 7 58 06 PM](https://user-images.githubusercontent.com/111031232/189256291-7f3928a7-80d9-46dd-a12c-bc183276129b.png)

I guess it remembered my previous sessions (maybe I starred and un-starred...), but I don't know.

**Comment by radar-blip** [comment, marked as answer]
Yes, apparently you did, at one time or another. 
Else, it wouldn't be there. 😕
